src/Camera/QGCCameraControl.cc

```diff
@@ -2020,6 +2036,26 @@ QGCCameraControl::_downloadFinished()
     //reply->deleteLater();
 }
 
+void QGCCameraControl::_ftpDownloadComplete(const QString& fileName, const QString& errorMsg)
+{
+    qCDebug(CameraControlLog) << "QGCCameraControl::_ftpDownloadComplete fileName:errorMsg" << fileName << errorMsg;
```

Comment:
You don't need to set class and function name, that can be configured  via environment variable:
https://doc.qt.io/qt-5/debug.html